Project.ConsoleApp.Test/ConsoleAppTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+
+
+namespace Project.ConsoleApp.Test
+{
+    [TestClass]
+    public class ConsoleAppTest
+    {
+        string _validJSONFile = string.Empty;
+        string _invalidJSONFile = string.Empty;
+        string _validXMLFile = string.Empty;
+        string _invalidXMLFile = string.Empty;
+
+        public ConsoleAppTest()
+        {
+            string testFilePath = Directory.GetCurrentDirectory() + @"\TestFiles\";
+
+            _validJSONFile = testFilePath + "JSON_ValidFormat.json";
```

Comment:
`Purposeful Code`: Since you aren't checking for the actual existence of a file, having these files really doesn't serve a purpose.   Really, an string value would suffice to pass a test.